common/primitives/src/payment.rs

```diff
@@ -0,0 +1,68 @@
+use frame_support::DefaultNoBound;
```

Comment:
I noticed that we are moving these to the primitives. However, I wonder if we could avoid this. The pallet consuming these types already needs to call functions as well. I appreciate the portability that the frequency transaction pallet currently offers. It might be beneficial to maintain these within the frequency-payment-pallet to keep the pallet more portable and reduce dependencies. What are your thoughts on this?